src/core/shared_memory_manager.h

```diff
@@ -59,6 +74,9 @@ class SharedMemoryManager {
     size_t byte_size_;
     int shm_fd_;
     void* mapped_addr_;
+    void* cuda_ipc_addr_;
```

Comment:
Don't really need this extra field, it serves the same purpose as `mapped_addr_`, you can differentiate the memory type by `kind_`

Reply:
Good point. Fixed.